spec/lib/saml_idp/controller_spec.rb

```diff
@@ -30,18 +30,28 @@ def params
       saml_response = encode_response(principal)
       response = OneLogin::RubySaml::Response.new(saml_response)
       response.name_id.should == "foo@example.com"
-      response.issuer.should == "http://example.com"
+      response.issuers.first.should == "http://example.com"
```

Comment:
apparently not. I searched awhile to see where ruby-saml dropped that method, but couldn't find it in the changelog.